content/docs/user-guide/gems/reference/aws/aws-gamelift/gem-setup.md

```diff
@@ -87,28 +87,21 @@ You must include the AWS GameLift Gem static library in your project's CMake bui
     )
     ```
 
-## 3. Integrate game
+## 3. Integrate game and dedicated server
 
-For a client application, your game must implement the following use cases to manage a session:
-- `CreateSession`
-- `SearchSessions`
-- `JoinSession`
-- `LeaveSession`
+Check [Session Management Integration](session-management/integration/) for managing game sesions within your game and dedicated server.
+To support the optional FlexMatch feature, check [FlexMatch Integration](flexmatch/integration/).
 
-The AWS GameLift Gem provides both [C++ APIs](cpp-api/) and [scripting](scripting/). You can implement these use cases using either method.
 
+## 4. Set up AWS credentials
 
-## 4. Integrate server
-
-To establish communication between your server and GameLift, you must notify GameLift that your server is ready, and then have your server respond to GameLift notifications.
-
-For more details on GameLift server notifications, refer to the [Server notifications](cpp-api/#server-notifications) section of the AWS GameLift Gem C++ API page.
+To work with AWS resources in O3DE you must set up AWS credentials for your users. 
 
-## 5. Set up AWS credentials
+For more details on configuring your AWS credentials in O3DE, refer to [Configuring AWS Credentials](/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials/).
 
-To work with AWS resources in O3DE you must set up AWS credentials for your users. 
+Please check [IAM policy examples for GameLift](https://docs.aws.amazon.com/gamelift/latest/developerguide/gamelift-iam-policy-examples.html) for adding proper permissions to use the Amazon GameLift service.
```

Comment:
"... to configure appropriate permissions required to use the GameLift service".